docs/Explanation/ADR/adr-004-did.md

```diff
@@ -0,0 +1,705 @@
+  "versionId": "35ba49b94b20ac0a1805aa5283035ea71b003f56e3aeda6a4e9027779fe4aef5", /* hex(blake2b(tx)) */ // <-- new hash computed
+  "deactivated": false
+}
+```
+
+###### Deactivate
+
+A DID can be deactivated using the gRPC message:
+
+```
+MsgDeactivateDid(did string, signerAccount string)
+```
+
+The DID identified by the parameter `did` MUST exist on-chain and the `signerAccount` MUST exist as a verification method (property `blockchainAccountID`) in a verification relationship of type `authentication` or being listed as a DID controller.
+
+This operation MUST remove all the verification methods and controllers and set the metadata property `deactivated` to true. This operation is not reversible.
+
+Example:
+
+```javascript
+/* gRPC message */
+MsgAddDeactivateDid(
+    "did:cosmos:cash:806e557e-ecdb-4e80-ab0d-a82ad35c9ceb", 
+    "cash1n90e4s33ljxn00lhucvnmnfjng773efma38dc2"
+)
+
+/* DID document */
+{
+    "context": [
+        "https://www.w3.org/ns/did/v1"
+    ],
+    "id": "did:cosmos:cash:806e557e-ecdb-4e80-ab0d-a82ad35c9ceb",
+    "controller": [], // <-- controllers are removed 
+    "verificationMethods" : [] // <-- verification methods are removed
+}
+
+/* DID metadata */
+{
+  "created": "2021-03-23T06:35:22Z",
+  "updated": "2021-09-23T06:35:22Z",  // <--  update field modified
+  "versionId": "e5ca728b93d19daa54180e20eec76d0d5614656c2ee3509df5a21a3abc1249ea", /* hex(blake2b(tx)) */ // <-- new hash computed
+  "deactivated": true  // <-- field deactivated set to true
+}
+```
+
+### Method-specific Properties 
+
+#### DID-core Verification Material
+
+The [Verification Material](https://www.w3.org/TR/did-core/#verification-material) type must be `EcdsaSecp256k1RecoveryMethod2020`.
+
+The content of the verification material can be `blockchainAccountID`, but for interoperability reasons, the verification material should support also `publicKeyHex`. 
+
+Support for other verification materials might be introduced. 
+
+### Verification Relationships
+
+The DID document MUST support the following [verification relationships](https://www.w3.org/TR/did-core/#verification-relationships):
+
+- [`authentication`](https://www.w3.org/TR/did-core/#authentication) - authorizes amends to the DID document
+- [`assertionMethod`](https://www.w3.org/TR/did-core/#assertion)
+- [`keyAgreement`](https://www.w3.org/TR/did-core/#key-agreement)
+- [`capabilityInvocation`](https://www.w3.org/TR/did-core/#capability-invocation)
+- [`capabilityDelegation`](https://www.w3.org/TR/did-core/#capability-delegation)
+
+
+##### DID Document Metadata
+
+The implementation for [DID document metadata](https://www.w3.org/TR/did-core/#did-document-metadata) MUST report the following properties for a DID document:
+
+- `created`: a [datetime](https://www.w3.org/TR/xmlschema11-2/#dateTime) string of the creation date that is the UTC date associated with the block height when the DID document was submitted the first time
```

Comment:
Would this benefit by stating the string format? e.g. [RFC3339](https://datatracker.ietf.org/doc/html/rfc3339)